src/test/regress/expected/multi_subquery_behavioral_analytics.out

```diff
@@ -546,62 +546,103 @@ ORDER BY user_lastseen DESC, user_id;
 
 ------------------------------------
 -- Count the number of distinct users_table who are in segment X and Y and Z
--- This query will be supported when we have subqueries in where clauses.
 ------------------------------------
-SELECT DISTINCT user_id
+SELECT user_id
```

Comment:
> i. Do we support distinct? Do we want to keep a regression with distinct?

Not, Citus does not support DISTINCT on the top level query at all.

> ii. Shall we group by another column other than user_id to show that we don't need to group by user_id?

I wanted to mimic `DISTINCT` via the `group by`. These queries are coming from `Raw Queries` document and I feel like sticking to those queries is slightly better for documentation?